src/Core/Managed/Shared/Extensibility/TelemetryConfiguration.cs

```diff
@@ -146,7 +146,35 @@ public IList<ITelemetryInitializer> TelemetryInitializers
         {
             get { return this.telemetryInitializers; }
         }
-        
+
+        /// <summary>
+        /// Gets or sets the first telemetry processor in the chain of processors.
+        /// </summary>
+        public ITelemetryProcessor TelemetryProcessor
```

Comment:
conceptually, TelemetryConfiguration contains a Chain of TelemetryProcessors not a TelemetryProcessor...but the code hides away the chain part and implicitly assumes that the reference to TelemetryProcessor here is "first" of the chain...
for someone configuring this in code, how do they do that?